tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py

```diff
@@ -108,3 +120,39 @@ def a11y_toggle_invert(self):
 
     def a11y_toggle_grayscale(self):
         self.a11y_toggle_switch(self.marionette.find_element(*self._grayscale_switch_locator))
+
+
+class AccessibilityAudio(Base):
+
+    _page_locator = (By.ID, 'accessibility-audio')
+
+    @property
+    def screen_element(self):
+        return self.marionette.find_element(*self._page_locator)
+
+
+class AccessibilityInput(Base):
+
+    _page_locator = (By.ID, 'accessibility-input')
+    _delay_locator =(By.CSS_SELECTOR, '[name="ui.click_hold_context_menus.delay"]')
+    _confirm_delay_change_locator =(By.CSS_SELECTOR, 'button[data-l10n-id="ok"]')
+
+    @property
+    def screen_element(self):
+        return self.marionette.find_element(*self._page_locator)
+
+    def tap_change_delay(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._delay_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        self.marionette.switch_to_frame()
+        Wait(self.marionette).until(expected.element_displayed(*self._confirm_delay_change_locator))
+
+    def tap_confirm_delay(self):
```

Comment:
At the moment I only take screenshots and do not do actual selection - once we expand the class to pick selection, we should put it in its own class.  I wonder whether there is a prototype class that we can create to use for system dialogs like this.